packages/web-runtime/src/App.vue

```diff
@@ -158,7 +154,7 @@ export default {
     },
 
     sidebarNavItems() {
-      if (this.publicPage()) {
+      if (!this.user.token) {
```

Comment:
What was the motivation behind this change here?

Reply:
`publicPage` is true for a page that can be used in both authorised/unauthorised context. So if the user would be in location picker with his own private resources he wouldn't see the navigation because it is a public page.